graviti/schema/base.py

```diff
@@ -183,3 +183,14 @@ def to_pyarrow(self) -> pa.DataType:
 
         """
         raise NotImplementedError
+
+    def get_keys(  # pylint: disable=no-self-use
+        self,
+    ) -> Union[Tuple[str, ...], List[Tuple[str, ...]]]:
```

Comment:
why the return type is a union of `List` and `Tuple`, can we return Tuple for all get_keys method?

Reply:
The return type of `record.get_keys()` is `List`, since there are multiple names in the record.